docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -0,0 +1,309 @@
+
+![Redshift Cluster Endpoint](../../../img/database-access/guides/redshift/cluster-endpoint@2x.png)
+
+You'll only need the `host:port` portion of the endpoint.
+
+- Database user name and database name.
+
+If you haven't created any database users, default database admin user name and
+maintenance database can be found on the `Database configurations` panel of the
+`Properties` tab:
+
+![Redshift Cluster User/Database](../../../img/database-access/guides/redshift/cluster-user-db@2x.png)
+
+## Create IAM Policy
+
+Teleport uses [IAM tokens](https://docs.aws.amazon.com/redshift/latest/mgmt/generating-user-credentials.html)
+to authenticate with Redshift databases. IAM authentication is enabled by default
+for all Redshift databases.
+
+In order to authorize Teleport to generate temporary IAM tokens, create an IAM
+role with the following permissions:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "redshift:GetClusterCredentials",
+            "Resource": [
+                "arn:aws:redshift:*:1234567890:dbuser:*/*",
+                "arn:aws:redshift:*:1234567890:dbname:*/*",
+                "arn:aws:redshift:*:1234567890:dbgroup:*/*"
+            ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": "redshift:DescribeClusters",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+The `DescribeClusters` permission is required to allow Teleport to retrieve
+cluster ID, region and port.
+
+The `GetClusterCredentials` permission controls access to generating IAM tokens.
+The resource ARN string has the following format:
+
+```
+arn:aws:redshift:<region>:<account-id>:<resource>:<cluster-id>/<name>
+```
+
+Parameters:
+
+- `region`: AWS region where Redshift cluster is deployed, or a wildcard.
+- `account-id`: AWS account ID the Redshift cluster is deployed in.
+- `resource`: One of `dbuser`, `dbname` or `dbgroup` to restrict access to
+  database accounts, names or groups respectively.
+- `cluster-id`: Redshift cluster identifier, or a wildcard.
+- `name`: Name of a particular database account, name or group (depending
+  on the `resource`), or a wildcard.
+
+<Admonition type="note" title="Users auto-create">
+  Teleport does not currently use auto-create option when generating tokens so
+  users must exist in the database.
+</Admonition>
+
+See [AWS documentation](https://docs.aws.amazon.com/redshift/latest/mgmt/generating-iam-credentials-role-permissions.html)
+for more information about setting up an IAM role.
+
+## Setup Teleport Auth and Proxy Services
+
+Teleport Database Access for Redshift is available starting from `6.2` release.
+Download the appropriate version of Teleport for your platform from our
+[downloads page](https://goteleport.com/teleport/download).
+
+Teleport requires a valid TLS certificate to operate and can fetch one automatically
+using Let's Encrypt [ACME](https://letsencrypt.org/how-it-works/) protocol.
```

Comment:
I get the following message when using two SSL certs (one per Node) from Let's Encrypt (generated via certbot) when trying to join one Node into the other's cluster:

```
Node failed to establish connection to cluster: Get "https://teleport.cluster.local/v2/domain": x509: certificate signed by unknown authority, Get "https://172.31.18.224:3025/webapi/find": x509: cannot validate certificate for 172.31.18.224 because it doesn't contain any IP SANs. service/connect.go:67
```

Can someone confirm? Is there a better alternative here if that's reproducible?

Reply:
Issue resolved! The IP address in the generated join command must be the public proxy. I've added that admonition elsewhere. Thanks!